docs/core/sdk.md

```diff
@@ -22,7 +22,7 @@ It contains the following components:
 
 1. The .NET Core Command Line Tools that are used to build applications
 2. .NET Core (libraries and runtime) that allow applications to both be built and run
-3. The `dotnet` driver for running the CLI commands as well as running applications
+3. The `dotnet` driver for running the [CLI commands](tools/index.md) as well as running applications
 
 
 ## Acquiring the .NET Core SDK
```

Comment:
yes, we're trying to clean those up, get better organized and clearly identify things what we think the community could help us out by using the up-for-grabs label: https://github.com/dotnet/core-docs/labels/up-for-grabs. I'll see if there's something we can add in the contributing guide to help with this as well.